bessctl/module.py

```diff
@@ -22,10 +22,18 @@ def __init__(self, **kwargs):
         else:
             name = None
 
-        ret = self.bess.create_module(self.__class__.__name__, name,
+        if(not _do_not_create):
+            #create an object
+            ret = self.bess.create_module(self.__class__.__name__, name,
                                       self.choose_arg(None, kwargs))
-
-        self.name = ret.name
+            self.name = ret.name
+        else:
+            #bind to a pre-existing object, check if it's real
+            assert name != None
+            info = self.bess.get_module_info(name)
+            assert info != None
```

Comment:
Checking `info` here doesn't do anything. It won't return `None` at all. Just either success or exception.